website/.env

```diff
@@ -0,0 +1,3 @@
+NEXT_PUBLIC_ALGOLIA_APP_ID=YY0FFNI7MF
+NEXT_PUBLIC_ALGOLIA_INDEX=product_VAGRANT
+NEXT_PUBLIC_ALGOLIA_SEARCH_ONLY_API_KEY=5037da4824714676226913c65e961ca0
```

Comment:
Safe to add to VC, according to https://www.algolia.com/doc/guides/security/api-keys/#search-only-api-key